docs/admin/access.rst

```diff
@@ -9,62 +9,52 @@ Access control
     specifically built for Weblate. If you are using anything older, please consult
     the documentation for the specific version you are using.
 
-Weblate comes with a fine-grained privilege system to assign user permissions
-for the whole instance, or in a limited scope.
+After installing Weblate, you can grant permissions to various things on the
+whole installation using :ref:`default-groups`. :ref:`acl` lets you grant
+permissions to specific translation projects. If you want to fine-tune the
+permissions you give based on groups you make yourself, you can do so
+using :ref:`custom-acl`.
 
-The permission system is based on groups and roles, where roles define a set of
-permissions, and groups assign them to users and translations, see
-:ref:`auth-model` for more details.
-
-After installation a default set of groups are created, and you can use those
-to assign users roles for the whole instance (see :ref:`default-groups`). Additionally when
-:ref:`acl` is turned on, you can assign users to specific translation projects.
-More fine-grained configuration can be achieved using :ref:`custom-acl`.
-
-Common setups
--------------
-
-Locking down Weblate
-++++++++++++++++++++
-
-To completely lock down your Weblate, you can use :setting:`REQUIRE_LOGIN` to
-force users to sign in and :setting:`REGISTRATION_OPEN` to prevent new
-registrations.
+In the permission system, roles define a set of permissions, and you can
+put users in groups that have one or more of these roles, described in
+:ref:`auth-model`.
 
 Site wide permissions
 +++++++++++++++++++++
 
-To manage permissions for a whole instance, just add users to `Users` (this is done
-by default using the :ref:`autogroup`), `Reviewers` and `Managers` groups. Keep
-all projects configured as `Public` (see :ref:`acl`).
```

Comment:
It isn't obvious _why_ all projects should be kept public?
Is it because permissions won't be site-wide if some projects are made private?

Reply:
It's relevant for a "Common setup" "Site wide permissions" i.e. if you want to manage permissions site-wide you should keep projects as `public`.

Reply:
It just looked like a general recommendation to me.